bedrock/base/templates/includes/global-nav.html

```diff
@@ -211,6 +211,11 @@ <h3 class="summary" data-id="technology">
                   </a>
                 </li>
                 <li>
+                  <a href="{{ url('mozorg.developer') }}" data-link-type="nav" data-link-name="Mozilla Developers" data-link-group="Technology">
+                    {{ _('Mozilla Developers') }}
```

Comment:
I don't know if we have the pages to warrant a subnav at this point. Maybe once we redesign the technology page?